sdk/identity/identity/src/credentials/managedIdentityCredential/msalMsiProvider.ts

```diff
@@ -163,7 +165,7 @@ export class MsalMsiProvider {
 
           if (result === null) {
             throw new CredentialUnavailableError(
-              "The managed identity endpoint was reached, yet no tokens were received.",
+              "Attempted to use the token exchange managed identity, but received a null token.",
```

Comment:
Is this slightly more accurate?
```suggestion
              "Attempted to use the token exchange managed identity, but received a null response.",
```